pipelines/tekton/gitops-update-pipeline/gitops-update-pipeline.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: gitops-update-pipeline
+spec:
+  params:
+    - name: gitServer
+      type: string
+    - name: gitOrgName
+      type: string
+    - name: gitRepoName
+      type: string
+    - name: imageDigest
+      type: string
+    - name: imageReferenceFilePath
+      type: string
+    - name: gitApiPrefix
+      default: /api/v1
+      type: string
+    - name: gitTokenSecretName
+      type: string
+    - name: gitTokenSecretKey
+      default: token
+      type: string
+  tasks:
+    - name: git-clone
+      params:
+        - name: url
+          value: $(params.gitServer)/$(params.gitOrgName)/$(params.gitRepoName)
+        - name: gitInitImage
+          value: registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8@sha256:1a50511583fc02a27012d17d942e247813404104ddd282d7e26f99765174392c
+        - name: subdirectory
+          value: /$(params.gitRepoName)/
+      taskRef:
+        kind: ClusterTask
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: git-workspace
+        - name: basic-auth
+          workspace: git-basic-auth
+    - name: commit-image-ref
+      params:
+        - name: BASE_IMAGE
+          value: cgr.dev/chainguard/git:root-2.39@sha256:7759f87050dd8bacabe61354d75ccd7f864d6b6f8ec42697db7159eccd491139
```

Comment:
Any particular reason for using this image btw?

Reply:
This is just the default image from the git-cli cluster task that comes with OpenShift Pipelines, so I think it got pulled out to a parameter with the same default value when creating the pipeline through the UI. Is there some other image that you want to suggest instead? If so, maybe that should be tackled as a separate issue where it can be tested.

Reply:
Is the `BASE_IMAGE` required since the [git-cli task](https://hub.tekton.dev/tekton/task/git-cli) already defines that image as the default value?

Reply:
Probably not, but I think the pipeline creation UI pulled that out to a corresponding parameter at the pipeline level, with the same default value; so while it _can_ be overridden by a user, it doesn't _need_ to be. Maybe this is useful for cases where there's a CVE in this image, or they want a newer version of Git?

OTOH, I wonder if this is actually unintentionally pinning the version to this digest, even if the underlying git-cli cluster task updates to a new image over time :thinking: